components/raftstore/src/store/fsm/peer.rs

```diff
@@ -3003,11 +3045,7 @@ where
                 self.on_voter_replicated_index_response(msg.get_extra_msg());
             }
             ExtraMessageType::MsgGcPeerRequest => {
```

Comment:
This message is only used by the 'CheckGcPeer' protocol in raftstorev2. I prefer not to mix them up, as raftstorev2 is not actively being developed and adding this would increase complexity.

Reply:
Agree, it also troubles me a lot. 

Meanwhile, Implementing a retry mechanism seems necessary in case messages are unexpectedly lost